### PR TITLE
[release] whole heaps of content updates

### PR DESCRIPTION
**Pull request by mbifulco**

### CODE REVIEW

File: _data/take5s/GYM-5023.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+course_ID: GYM-5023
+title: "Creating A Harmonious Color Scheme with HSL"
+date: 2020-01-13T00:00:00-04:00
+course_type: take5
+url: /static/take5/GYM-5023/
+poster_art: /img/take5/posters/gym-5023.jpg
+live: true
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5023.yml]: This should be `false` — it should only be live on 1/13

File: _data/take5s/GYM-5023.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+instructor: "Jeremy Osborn"
+topic: "Web Design & Development"
+video_ID: 1430bmqJ9xw
+video_duration: "4:57"
+featured: true
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5023.yml]: featured should be `false`

**Review by andrewpmiller — CHANGES_REQUESTED** [review]
GYM-5023 needs a couple of edits so that it doesn't go live ahead of schedule.

**Review by jgagne — APPROVED** [review]

**Review by andrewpmiller — APPROVED** [review]
👍

**Pull request merged**